test/shell/t1.sh

```diff
@@ -9,7 +9,13 @@
 set -o errexit
 set -o pipefail
 
-timeout --kill-after=3s 1s ./mgmt --help # hello world!
+if [[ $(uname) == "Darwin" ]] ; then
```

Comment:
Could do, I didn't want to just go ahead and shove it into shared code. Especially since the test are still failing.